### DISCUSSION TITLE
LuaJava supports java 11+ ?

### DISCUSSION THREAD

**Discussion by skinkairewalker** [gudzpoz/luajava#147] [General]
hello everyone !!
its possible use luajava with Java 11 or highter ?

**Comment by gudzpoz** [comment]
The test cases all pass on Java 11. But depending on your use cases, if you want to access some other modules with Lua, you might need to customize `module-info.java` to allow reflective access to it.

See https://gudzpoz.github.io/luajava/jpms.html and https://en.wikipedia.org/wiki/Java_Platform_Module_System .
